src/terminal/Capabilities.cpp

```diff
@@ -163,6 +163,7 @@ namespace
         String { "ec"_tcap, "ech"sv, "\033[%p1%dX"sv },
         // Clear to end of screen
         String { "cd"_tcap, "ed"sv, "\033[J"sv },
+        String { "E3"_tcap, "E3"sv, "\033[3J"sv },                    // Defines the E3 capability
```

Comment:
It took me a while to find out where this `E3` capability is coming from, but looks like from this quote:
```
# E3 clears the terminal's scrollback buffer.  This was implemented in the
#    Linux 3.0 kernel as a security feature.  It matches a feature which was
#    added in xterm patch #107.
```
`E3` capability was first introduced by the linux kernel (version 3.0). I guess they're talking about the builtin tty layer there.